compiler-core/src/pretty.rs

```diff
@@ -39,6 +39,12 @@ pub trait Documentable<'a> {
     fn to_doc(self) -> Document<'a>;
 }
 
+impl<'a> Documentable<'a> for bool {
+    fn to_doc(self) -> Document<'a> {
+        Document::Str(if self { "true" } else { "false" })
```

Comment:
Sorry, one last thing. Please remove this as bools do not have a universal format.

Reply:
I will do it!